### PR TITLE
[Merged by Bors] - Windows build fix 

### PR DESCRIPTION
**Pull request by moshababo**
Windows build fix:
* use `protobuf` /  `grpc-gateway` versions as specified in `go.mod`
* fix small typo in `Makefile`

### CODE REVIEW

File: setup_env.bat [hunk 1]
```diff
@@ -4,8 +4,19 @@ if %ERRORLEVEL% GTR 0 exit /B 1
 
 call ./scripts/win/install-protobuf.bat
 
-go get github.com/golang/protobuf/protoc-gen-go
-set GO111MODULE=off
-go get github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
-go get github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
-set GO111MODULE=on
+SET GO111MODULE=on
```
Comment (noamnelke) [inline setup_env.bat]: This is not needed, since it should be `on` to begin with. The previous code turned it `off` and back `on`.

**Review by noamnelke — APPROVED** [review]
Only semi-related, and still: can you please cleanup `setup_env.sh`, too, by removing all the commented out lines there (they're all not relevant anymore).

**Comment by noamnelke** [comment]
Also, for some reason I don't understand `go fmt ./...` results in changes, so you should run that I guess?

**Comment by noamnelke** [comment]
The `fmt` thing is very strange. I can't reproduce the change locally, must be something Travis-related. Tried to re-run in Travis now 🧐

**Comment by noamnelke** [comment]
Still fails. Maybe due to different go patch version? Can you please investigate?

**Comment by noamnelke** [comment]
Travis issue was fixed by deleting caches 🤷🏻‍♂️

**Comment by moshababo** [comment]
bors merge

**Comment by moshababo** [comment]
bors merge

**Pull request closed without merging**